packages/typescript/internal/ts_project.bzl

```diff
@@ -642,25 +635,18 @@ def ts_project_macro(
                 # but that's our own code, so we don't.
                 "@npm//protobufjs",
                 # END-INTERNAL
-                Label("//packages/typescript/internal/worker:worker"),
-                Label(worker_tsc_bin),
-                Label(worker_typescript_module),
+                Label(typescript_package),
+                Label("//packages/typescript/internal/worker:filegroup"),
                 tsconfig,
             ],
             entry_point = Label("//packages/typescript/internal/worker:worker_adapter"),
             templated_args = [
-                "$(execpath {})".format(Label(worker_tsc_bin)),
-                "--project",
-                "$(execpath {})".format(tsconfig),
-                # FIXME: should take out_dir into account
```

Comment:
I think these FIXMEs still apply?

Reply:
No, we now get the same parameters that are passed to the non-worker tsproject variant. Before we were reading startup args, but now we use the worker args